packages/excalidraw/scene/Scene.ts

```diff
@@ -299,6 +300,32 @@ class Scene {
     this.elements = syncInvalidIndices(_nextElements);
     this.elementsMap.clear();
     this.elements.forEach((element) => {
+      // NOTE (mtolmacs): This is a temporary fix for very large scenes
+      if (
+        Math.abs(element.x) > 1e6 ||
+        Math.abs(element.x) > 1e6 ||
+        Math.abs(element.width) > 1e6 ||
+        Math.abs(element.height) > 1e6 ||
+        // @ts-ignore
+        element.points?.some(
+          (point: LocalPoint) =>
+            Math.abs(point[0]) > 1e6 || Math.abs(point[1]) > 1e6,
+        )
+      ) {
+        console.error(
+          `replaceAllElements() received an element with invalid dimensions`,
+          element.id,
+          element.x,
+          element.y,
+          element.width,
+          element.height,
+          // @ts-ignore
+          element.startBinding?.elementId,
+          // @ts-ignore
+          element.endBinding?.elementId,
+        );
+      }
+
```

Comment:
isn't this gonna obliterate our Sentry as it will be happening crazy often on compromised scenes?